src/jit/compiler.cpp

```diff
@@ -49,6 +49,13 @@ AssemblyNamesList2* Compiler::s_pAltJitExcludeAssembliesList            = nullpt
 #endif // ALT_JIT
 
 /*****************************************************************************/
+#if defined(_TARGET_AMD64_) && defined(_MSC_VER)
+#include <intrin.h>
```

Comment:
I copied (and tweaked) the stuff from gc.cpp suggested earlier - would you mind taking another look, @BruceForstall ? Thanks. And yes, you're absolutely right - we're measuring this code running on the host, it doesn't matter what it's targeting. Mea culpa.